testapps/integration/web/index.php

```diff
@@ -29,12 +29,12 @@
 });
 
 $app->post('/logging_standard', function (Request $request) {
-    $logName = $request->get('log_name');
-    $token = $request->get('token');
+    $logName = $request->request->get('log_name');
```

Comment:
It looks like this isn't provided to the logging_standard test anymore, but it is to the logging_custom test.